evalml/tests/data_checks_tests/test_data_checks.py

```diff
@@ -352,3 +354,25 @@ def validate(self, X, y):
         "errors": [],
         "actions": [DataCheckAction(DataCheckActionCode.DROP_COL, metadata={"column": 'col_to_drop'}).to_dict()]
     }
+
+
+def test_data_checks_drop_index(X_y_binary):
+    X, y = X_y_binary
+    X = pd.DataFrame(X)
+    X['index_col'] = pd.Series(range(len(X)))
+    X = ww.DataTable(X)
+    X = X.set_index('index_col')
+
+    class MockDataCheck(DataCheck):
+        def validate(self, X, y):
+            return {"warnings": [], "errors": [], "actions": []}
+
+    assert MockDataCheck().validate(X, y)
```

Comment:
is this assert necessary? Can we just assert that it equals the dict of empty warnings/errors/actions?

Reply:
I needed it for codecov to pass or else it would fail on `return {"warnings": [], "errors": [], "actions": []}` 😄